xds/src/main/java/io/grpc/xds/internal/security/trust/XdsTrustManagerFactory.java

```diff
@@ -50,6 +51,9 @@
 public final class XdsTrustManagerFactory extends SimpleTrustManagerFactory {
 
   private static final Logger logger = Logger.getLogger(XdsTrustManagerFactory.class.getName());
+  @VisibleForTesting
+  static boolean enableSpiffe = GrpcUtil.getFlag("GRPC_EXPERIMENTAL_SPIFFE_TRUST_BUNDLE_MAP",
```

Comment:
Delete this. With the flag in the file provider there won't be a trust bundle, so the new constructor won't be called.